server/tests/services/queue.js

```diff
@@ -2,20 +2,28 @@ import queueService from '../../services/queue';
 import queueManager from '../../managers/queue';
 import constants from '../constants';
 
-const ONE_MINUTE_MILLI = 60 * 1000;
+const ONE_MINUTE_MILLISECOND = 60 * 1000;
 
 describe('## Queue Service', () => {
   before(() => queueManager.enterTestMode());
   afterEach(() => queueManager.clearTestMode());
 
   describe('# createDelayedPage()', () => {
     it('creates a delayed page', (done) => {
-      queueService.createDelayedPage(constants.ticket, constants.user, 0)
-        .then(() => {
+      const title = constants.ticket.metadata.title;
+
+      queueService.createDelayedPage(
+        constants.ticket.id,
+        constants.user.id,
+        constants.user.devices[0],
+        1 * ONE_MINUTE_MILLISECOND,
+        constants.ticket.metadata.title).then(() => {
           expect(queueManager.testMode.jobs.length).to.equal(1);
           expect(queueManager.testMode.jobs[0].type).to.equal('page');
-          expect(queueManager.testMode.jobs[0].data.title).to.equal(constants.ticket.metadata.title);
-          expect(queueManager.testMode.jobs[0]._delay).to.equal(constants.user.delays[0] * ONE_MINUTE_MILLI);
+          expect(queueManager.testMode.jobs[0].data.title)
+            .to.equal(title + ' - ' + constants.user.devices[0].name);
```

Comment:
If this logic gets repeated in one more place, I'd like to see it removed into some kind of helper function, to DRY up the code.